tests/functional/vault/test_strategies.py

```diff
@@ -20,7 +20,7 @@ def test_addStrategy(web3, gov, vault, strategy, rando):
     with brownie.reverts():
         vault.addStrategy(strategy, 1000, 10, 50, {"from": rando})
 
-    assert vault.strategies(strategy) == [0, 0, 0, 0, 0, 0, 0]
+    assert vault.strategies(strategy) == [0, 0, 0, 0, 0, 0, 0, 0]
```

Comment:
With so many params, should we switch to comparing dicts?

Reply:
See #76 (which I didn't know existed until today, lol)